edge-modules/TestAnalyzer/Controllers/TwinStatusController.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer.Controllers
+{
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc;
+
+    [Route("api/[controller]")]
+    [ApiController]
+    public class TwinStatusController : Controller
+    {
+        // POST api/twinstatus
+        [HttpPost]
+        public async Task Post(ResponseStatus twinStatus)
```

Comment:
similar to DirectMethodStatusController, use Task<StatusCodeResult> as return type.

Reply:
I'm guessing you mean StatusCodeResult. Will do.